### ISSUE TITLE
Accessibility Checks Homepage: Errors

### ISSUE THREAD

**Issue by RedRoxProjects** [SamsungInternet/siteblog#31]
**What is the issue for?** 

Testing against accessibility - this issue highlights all the Errors against accessibility standards WCAG2AAA 

**Describe the task**
Correct all errors listed in this Issue

Errors: 
- [ ] Developer Hub - Page title is not logically nested - should be H1
- [ ] Feature Image - missing alt text
- [ ] Author Images - all missing alt text

**Additional context**
Errors highlighted by HTML_CodeSniffer

**Checklist before submitting**
- [x] I have checked the list of issues (open and closed) on this repository and this issue has not been reported.
- [x] I have selected the **DevHub Issue Board** under the **Projects** assignment (right of the page).
